widgetssdk/src/main/java/com/glia/widgets/core/visitor/GliaVisitorException.java

```diff
@@ -0,0 +1,26 @@
+package com.glia.widgets.core.visitor;
+
+import com.glia.androidsdk.GliaException;
+
+public class GliaVisitorException extends RuntimeException{
```

Comment:
This class is almost a duplicate of the `GliaException`. Also, I think it would be better to use it instead of this one there is no need in my opinion to have a dedicated Exception. Or if you want to wrap it to have a separate exception for Widget SDK then it might have a more generic name. For example `GliaWidgetException`